### PR TITLE
Texture info empty texure is null

### PR DESCRIPTION
**Pull request by timmmeh**
### Reference Issue

Adds test and fixes discussed in #2566

### CODE REVIEW

File: packages/model-viewer/src/features/scene-graph/material.ts [hunk 1]
```diff
@@ -36,9 +36,9 @@ const $backingThreeMaterial = Symbol('backingThreeMaterial');
 export class Material extends ThreeDOMElement implements MaterialInterface {
   private[$pbrMetallicRoughness]: PBRMetallicRoughness;
 
-  private[$normalTexture]: TextureInfo|null = null;
-  private[$occlusionTexture]: TextureInfo|null = null;
-  private[$emissiveTexture]: TextureInfo|null = null;
```
Comment (elalish) [inline packages/model-viewer/src/features/scene-graph/material.ts:41]: 👍

File: packages/model-viewer/src/test/index.ts [hunk 2]
```diff
@@ -51,6 +51,7 @@ import './features/scene-graph/model-spec.js';
 import './features/ar-spec.js';
 import './features/scene-graph/texture-spec.js';
 import './features/scene-graph/material-spec.js';
+import './features/scene-graph/texture-info-spec.js';
```
Comment (elalish) [inline packages/model-viewer/src/test/index.ts:54]: Did you forget to commit `texture-info-spec`?

Comment (timmmeh) [reply packages/model-viewer/src/test/index.ts:54]: oops

**Review by elalish — APPROVED** [review]
Looks good, thanks!

**Comment by elalish** [comment]
I just merged disabling the broken test runner. Please pull master and let's see if the tests pass now.

**Review by elalish — APPROVED** [review]
Thanks!

**Pull request merged**